app/scripts/modules/core/pipeline/config/triggers/webhook/webhookTrigger.module.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.core.pipeline.trigger.webhook', [
+    require('../../../../../core/config/settings.js'),
+  ])
+  .config(function (pipelineConfigProvider) {
+    pipelineConfigProvider.registerTrigger({
+      label: 'Webhook',
+      description: 'Executes the pipeline on a Webhook POST',
+      key: 'webhook',
+      controller: 'WebhookTriggerCtrl as ctrl',
+      controllerAs: 'vm',
+      templateUrl: require('./webhookTrigger.html'),
+      popoverLabelUrl: require('./webhookPopoverLabel.html'),
+    });
+  })
+  .controller('WebhookTriggerCtrl', function (trigger, $scope) {
+
+    $scope.trigger = trigger;
+
+    /* If this is a new trigger and constraints have not been set
+        yet, create an map to hold them in. */
+    if ($scope.trigger.constraints == null) {
+      $scope.trigger.constraints = {};
+      /* TODO: Find a way to add actual application and pipeline names into the constraints */
+      $scope.trigger.constraints['application'] = 'Application placeholder: FIX ME!';
```

Comment:
try `$scope.application.name` (it's available on the parent scope, which is the prototype of this `$scope` object)